documentation/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['../dev/eslint/rules/base_rules.js'],
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
+    node: true,
+    mocha: true,
+    jest: true,
+  },
+  parser: '@typescript-eslint/parser',
+  plugins: ['ban', 'filenames', 'local', '@typescript-eslint'],
+  settings: {},
+  rules: {
```

Comment:
I'm not an expert in this, but I think eslint merges configuration files together up to the root, so I don't think you have specify anything at all in this file other than the new or modified rules.